README.md

```diff
@@ -12,3 +12,19 @@ Almond is part of Open Thing Platform, a research project led by
 prof. Monica Lam, from Stanford University.  You can find more
 information at <https://thingpedia.stanford.edu/about>.
 
+## To Run
```

Comment:
If we bless docker, we should probably point people to our dockerhub repos and publish images there.

Reply:
Good idea, will do. Do we already have an account